data_steward/bq_utils.py

```diff
@@ -25,7 +25,8 @@ class InvalidOperationError(RuntimeError):
     """Raised when an invalid Big Query operation attempted during the validation process"""
 
     def __init__(self, msg):
-        super(InvalidOperationError, self).__init__(msg)
+        self.msg = msg
```

Comment:
Is this change related? I don't quite follow why it's needed and I don't see it referenced below.

Reply:
Ah forgot to mention it in my commit message- I put this for the assertRaises check [here](https://github.com/all-of-us/curation/pull/354/commits/c6cf95c3c936e4f1f53797ac9a3b74c6e9ef58be#diff-6cb8929a23157cc412569417bb428efaR457).